InvenTree/InvenTree/mixins.py

```diff
@@ -34,6 +34,31 @@ def update(self, request, *args, **kwargs):
 
         return Response(serializer.data)
 
+    def clean_string(self, data: str) -> str:
+        """Clean / sanitize a single input string.
+
+        Note that this function will *allow* orphaned <>& characters,
+        which would normally be escaped by bleach.
+
+        Nominally, the only thing that will be "cleaned" will be HTML tags
+
+        Ref: https://github.com/mozilla/bleach/issues/192
+        """
+
+        cleaned = clean(data, strip=True)
```

Comment:
Question: This adds additional stripping - won't that remove spaces that were formally left in? Are we creating a new situation like the current one with this in the future when someone uses spaces to signal *something*?

Reply:
I think that `strip` here is culling bad tags, not removing spaces.

e.g. something like:

| Original | strip=False | strip=True |
| --- | --- | --- |
| ```hello world <img src='hax0r.png'>``` | ```hello world &lt;img src='hax0r.png'&gt;``` | ```hello world ``` |